acp/acp_local.go

```diff
@@ -104,10 +106,14 @@ func (l *ACPLocal) Start(ctx context.Context) error {
 }
 
 func (l *ACPLocal) Close() error {
-	return l.manager.Terminate()
+	if !l.closed {
+		l.closed = true
+		return l.manager.Terminate()
```

Comment:
suggestion: Should probably `l.manager.Terminate()` first. Then if no error occurs do `l.closed = true`?